lib/kitchen/verifier/busser.rb

```diff
@@ -46,7 +46,7 @@ class Busser < Kitchen::Verifier::Base
         if verifier.windows_os?
           "$env:systemdrive\\opscode\\chef\\embedded\\bin"
         else
-          "/opt/chef/embedded/bin"
+          verifier.remote_path_join(%W[#{verifier[:chef_omnibus_root]} embedded bin])
```

Comment:
It didn't because a) the verifier object at this point has no knowledge of the config object and b) that's only set on the provisioner config object... :(

Unless there's some magic in the config_munger that @fnichol put in there :)